helper/funcs.go

```diff
@@ -18,6 +20,25 @@ func IsUUID(str string) bool {
 	return validUUID.MatchString(str)
 }
 
+// HashUUID takes an input UUID and returns a hashed version of the UUID to
+// ensure it is well distributed.
+func HashUUID(input string) (output string, hashed bool) {
+	if !IsUUID(input) {
```

Comment:
We want to ensure that the UUID reported by the host has a certain amount of entropy (not 1234, etc)